src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -386,6 +387,7 @@ export default class DefaultAudioVideoController
             retryCount: this.totalRetryCount,
             signalingOpenDurationMs: this.meetingSessionContext.signalingOpenDurationMs,
             iceGatheringDurationMs: this.meetingSessionContext.iceGatheringDurationMs,
+            startDurationMs: Date.now() - this.meetingSessionContext.startAudioVideoTimestamp,
```

Comment:
You should include the meeting-start duration in other "end" events too. e.g., When the meeting has ended with or without an error, builders can see the starting duration too.
- `meetingEnded`
- `meetingFailed`

Reply:
Included...